scripts/deploy-kubevirt-and-cdi.sh

```diff
@@ -16,6 +16,12 @@ KUBEVIRT_NAMESPACE=${1:-kubevirt}
 kubectl apply -n "${KUBEVIRT_NAMESPACE}" -f "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-operator.yaml"
 kubectl apply -n "${KUBEVIRT_NAMESPACE}" -f "https://github.com/kubevirt/kubevirt/releases/download/${KUBEVIRT_VERSION}/kubevirt-cr.yaml"
 
+kubectl patch -n "${KUBEVIRT_NAMESPACE}" kubevirt kubevirt --type='json' -p '[{
+  "op": "add",
+  "path": "/spec/configuration/developerConfiguration/featureGates/-",
+  "value": "VMPersistentState",
+}]'
+
```

Comment:
Could you please explain why this is required? According with https://kubevirt.io/user-guide/compute/persistent_tpm_and_uefi_state/#backend-storage a `vmStateStorageClass` should be also provided.

Reply:
@jcanocan, it turns out that it isn't necessary. I initially added it because some Windows tests were raising concerns, but upon closer inspection, they weren't actually failing.
Great catch! I've removed the code.

Reply:
Thanks for the explanation. Happy to help :)